src/profile-logic/process-profile.js

```diff
@@ -1532,6 +1547,8 @@ export function processGeckoProfile(geckoProfile: GeckoProfile): Profile {
     counters,
     profilerOverhead,
     threads,
+    profilingLog,
+    profileGatheringLog,
```

Comment:
The bugzilla bug mentions adding them to `meta` instead of the root. What do you think?

Reply:
That was my initial thinking, but I implemented them to top-level, so I can write them at the very end of generating the JSON in each process (in case the JSON generation itself produces more logging). I'll update the bug title...

Reply:
Yeah, but I meant that we can still have it in ’meta’ in the processed format.

Reply:
I've kept it in the root for now to keep the gecko and processed formats in sync. But let me know if you feel strongly about it, we can change the processing afterwards too (probably without the need of an upgrader)